### DISCUSSION TITLE
Release 2.2.0

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#636] [Announcements]
**New Features:**
- New graphical Winetrick UX
- Programs can now be manually pinned

**Quality of Life:**
- Updates to Korean, French, and Polish locales
- Cleanup of program loading and fixes to pins

**Bug Fixes:**
- Fixed issues with the Rosetta 2 installer

**New Contributors:**
* @Alanimdeo made their first contribution in https://github.com/Whisky-App/Whisky/pull/601
* @GabCoolDude made their first contribution in https://github.com/Whisky-App/Whisky/pull/602
* @mavethee made their first contribution in https://github.com/Whisky-App/Whisky/pull/607
* @HeySora made their first contribution in https://github.com/Whisky-App/Whisky/pull/620
* @PortableProgrammer made their first contribution in https://github.com/Whisky-App/Whisky/pull/608

**Full Changelog:** https://github.com/Whisky-App/Whisky/compare/v2.1.3...v2.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Whisky-App/Whisky/releases/tag/v2.2.0'>Release 2.2.0</a>.</em>

**Comment by biluochangan** [comment]
Why did I only display a string of garbled code after downloading?

**Comment by hen12343** [comment]
I just downloaded this for the first time, and when I first opened it a window popped up and said Dependencies setup, I have Rosetta installed but not Gptk, so I clicked next and it said downloading Gptk but it got stuck saying zeroKB of zeroKB, any help?

**Comment by HermanGalimov** [comment]
Hey! I was waiting for the update because I thought I could fix the crashes. The app has started crashes every time I open it, even after update. Please help me to fix it. 
[Crashing report.txt](https://github.com/Whisky-App/Whisky/files/13463423/Crashing.report.txt)

**Reply by IsaacMarovitz** [reply]
Looks like one of your pinned applications is causing the crash when we try to extract the Icon. Could you provide the `Metadata.plist` from your bottle in `~/Library/Containers/com.isaacmarovitz.Whisky/Bottles/<BOTTLE ID>/Metadata.plist`

**Reply by HermanGalimov** [reply]
[metadata.txt](https://github.com/Whisky-App/Whisky/files/13464729/metadata.txt)

**Reply by HermanGalimov** [reply]
Is it possible to help me with that?

**Reply by HermanGalimov** [reply]
Hey! Can you help with my problem please?

**Comment by Evangello** [comment]
Error “App not supported on this Mac”, with grey cross over icon.
Sonoma 14.1

**Reply by IsaacMarovitz** [reply]
Whisky is only supported on Apple Silicon Macs.

**Comment by SzhIIH** [comment]
Just finished the update, and next to Naraka: Infinite, I can't play
